actors.md

```diff
@@ -56,6 +56,30 @@ Parameters:
 Return: None
 
 ```go
+func shouldSlash(block1, block2 BlockHeader) bool {
+     // First slashing condition, blocks have the same ticket round
+     if sameTicketRound(block1, block2) {
+     	return true
+     }
+
+     // Second slashing condition, miner ignored own block when mining
+     // Case A: block2 could have been in block1's parent set but is not
+     block1ParentTipSet := parentOf(block1)
+     if !block1Parent.contains(block2) &&
+     	block1ParentTipSet.Height == block2.Height &&
+	block1ParentTipSet.ParentCids == block2.ParentCids {
+	return true	
+     }
+     
+     // Case B: block1 could have been in block2's parent set but is not
+     block2ParentTipSet := parentOf(block2)
+     if !block2Parent.contains(block1) &&
+     	block2ParentTipSet.Height == block1.Height &&
+	block2ParentTipSet.ParentCids == block1.ParentCids {
+	return true	
+     }
+}
```

Comment:
Does Go return false by default if you don't specify?

Reply:
no I'm dumb